app/Services/Shared/Import/Routine/ApiSubmitter.php

```diff
@@ -359,6 +359,9 @@ private function replaceMappings(array $line): array
                         app('log')->debug(sprintf('Replaced source name "%s" with a reference to account id #%d', $source, $this->mapping[0][$source]));
                     }
                 }
+                if (empty(trim($transaction['description']))){
```

Comment:
I would use:

```php
if('' === trim((string)($transaction['description'] ?? '')) {
```

`empty()` will return true when the value is `0` or `"0"`, but without `empty()` you need to catch the (tiny) chance that `['description']` isn't set.

Sounds good?